src/utils/utils.py

```diff
@@ -33,6 +33,90 @@
 _MOSES_DETOKENIZER = MosesDetokenizer()
 
 
+def check_for_previous_checkpoints(serialization_dir, tasks, phase, load_model):
+    """
+    Check if there are previous checkpoints. 
+    If phase == target_train, we loop through each of the tasks from last to first 
+    to find the task with the most recent checkpoint. 
+    If phase == pretrain, we check if there is a most recent checkpoint in the run 
+    directory. 
+
+    Parameters 
+    ---------------------
+    serialization_dir: str, 
+    tasks: List of SamplingMultiTask objects, 
+    phase: str 
+    load_model: bool
+
+    Returns
+    ---------------------
+    task_directory: None or str
+    max_epoch: int, -1 if not found
```

Comment:
Why not None?

Reply:
Just because usually when it comes to these types of ints, convention is usually -1 if not found.